src/text/non_blocking/raw_text_reader.rs

```diff
@@ -1175,7 +1203,7 @@ mod reader_tests {
     #[case(" {{\"hello\"}} ", TextValue::Clob(Vec::from("hello".as_bytes())))]
     fn test_read_single_top_level_values(#[case] text: &str, #[case] expected_value: TextValue) {
         let reader = &mut RawTextReader::new(text);
-        reader.stream_complete();
+        // reader.stream_complete();
```

Comment:
Dangling comments below?

Reply:
One day we will have the [technology](https://github.com/rust-lang/rust-clippy/issues/1348) .. >.>